src/models/Scholarships.ts

```diff
@@ -15,6 +16,8 @@ interface ScholarshipData {
   year?: string;
   authorId?: string;
   authorEmail?: string;
+  states: string[];
+  eligibility?: ScholarshipEligibility;
```

Comment:
Hmm, I think it may be better to just have the interface type here instead of creating a custom class. It keeps things simpler when converting from/to Firestore as our design doesn't currently call for any custom objects here.